modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New optional `defaultExpandedIds` and `onToggleExpand` options to the `tree` component config. #TINY-9653
 - New optional `defaultSelectedId` option to the `tree` component config. #TINY-9715
 - New `accordion` plugin with the `InsertAccordion` command. #TINY-9730
+- Accessibility features to the `accordion` plugin. #TINY-9731
```

Comment:
Since this has not bee released yet you don't need to add a changelog item.

- [ ] Remove changelog item